docs/tutorial.rst

```diff
@@ -21,23 +26,40 @@ Due to an issue regarding TLS support in the Python included in macOS it is nece
 
     $ brew install python
 
+Windows
+^^^^^^^
+
+#. Install Python 2: https://www.python.org/downloads/release/python-2714/
+
+#. Update Environment Path to include Python root and Scripts folder.  
+  **Update system path to include Python 2.7 install folders (C:\Python27 and C:\Python27\Scripts)
```

Comment:
@cdcarter 
When viewing the tutorial.rst it views as 
**Update system path to include Python 2.7 install folders (C:Python27 and C:Python27\Scripts)

The backslash after C: is missing \ when viewing the rst in a browser.